### ISSUE TITLE
Sweep: Cannot initialize VectorStoreIndex on examples

### ISSUE THREAD

**Issue by TomPenguin** [run-llama/LlamaIndexTS#60]
### Details

I tried [this example here](https://github.com/run-llama/LlamaIndexTS/blob/ab886a34d7de091ba9e74f5a4502f95b92f62200/examples/storageContext.ts), but when I run it the first time, I get the following error,

```bash
TypeError: node.node.getContent is not a function
    at [ABSOLUTE_PATH_TO_YOUR_PROJECT]/node_modules/llamaindex/dist/index.js:1812:29
    at Array.map (<anonymous>)
    at ResponseSynthesizer.<anonymous> ([ABSOLUTE_PATH_TO_YOUR_PROJECT]/node_modules/llamaindex/dist/index.js:1811:30)
    at Generator.next (<anonymous>)
    at [ABSOLUTE_PATH_TO_YOUR_PROJECT]/node_modules/llamaindex/dist/index.js:71:61
    at new Promise (<anonymous>)
    at __async ([ABSOLUTE_PATH_TO_YOUR_PROJECT]/node_modules/llamaindex/dist/index.js:55:10)
    at ResponseSynthesizer.synthesize ([ABSOLUTE_PATH_TO_YOUR_PROJECT]/node_modules/llamaindex/dist/index.js:1810:12)
    at RetrieverQueryEngine.<anonymous> ([ABSOLUTE_PATH_TO_YOUR_PROJECT]/node_modules/llamaindex/dist/index.js:1843:39)
    at Generator.next (<anonymous>)
```

and the second time I run it, I get the following error.

```bash
Error: Cannot initialize VectorStoreIndex with both nodes and indexStruct
    at Function.<anonymous> ([ABSOLUTE_PATH_TO_YOUR_PROJECT]/node_modules/llamaindex/dist/index.js:3119:17)
    at Generator.next (<anonymous>)
    at fulfilled ([ABSOLUTE_PATH_TO_YOUR_PROJECT]/node_modules/llamaindex/dist/index.js:58:24)
```

I have initialized and executed the procedure as per the instructions, but is there any other step I need to take?

**Comment by TomPenguin** [comment]
Wow, sweep.ai is so cool.

**Comment by TomPenguin** [comment]
This issue has been resolved in the latest version (v0.0.17) and is closed.
